until.ts

```diff
@@ -0,0 +1,16 @@
+import curryN from './utils/curry_n.ts'
+import { Func, Curry3, Predicate1 } from './utils/types.ts'
+
+function _until(pred: Predicate1, fn: Func, init: any) {
```

Comment:
fn is a function of arity 1, so use `FuncAry1` defined in `types.ts`

Reply:
Ok

Reply:
its FuncArr1 i guess??